src/styles/button.scss

```diff
@@ -27,6 +27,7 @@
   width: 100%;
   padding: 10px 0;
   color: $color-white;
+  transition: width 2s, background-color 2s;
```

Comment:
Remember, this is going to apply the styles to ALL button components. If this is the desired effect, then ok. 

If you want to apply a transition at certain circumstances, apply the transition through a button story. 

Or check out the Animations stories, and wrap the button component with an Animation component like it's developed for. Whichever fits your end goal is